### DISCUSSION TITLE
Enable batch editing of description

### DISCUSSION THREAD

**Discussion by DamageIncorporated** [ente-io/ente#3017] [Enhancements]
When I import new their photos to albums,  I like to enter at least some basic description to the whole batch, for example "trip to the zoo".

At the moment, I have to open every single photo to paste "trip to the zoo" into it's description field. That means for 30 photos 3 clicks each plus pasting the string ctrl-v.

My wish for the future would be:

- select the whole batch, as i would do to throw them into an album (or the trash)
- click an ( i ) button on the toolbar (as I would do for a single photo)
- click the description field and enter "trip to the zoo" one time  for all photos
- click a checkmark to write the new description for the whole batch to the database.

Caveats:
- The fields with identical or empty value could be edited like outlined above.
- If at least one photo has a differing description from the rest, batch editing should be disabled to prevent accidental overwriting.
- If the description field is **prepopulated identically for all photos**, so should be my editing dialog to be able to modify (or delete) the description for the whole batch

- _Ideally_, this approach could be used for all other editable info fields
- **In Paradise** I would be able to batch edit _EXIF data_ of the photo-*files* in the same way, even if those don't end up in ente's database

**Comment by jbelien** [comment]
I was looking how to use categories and (if I understood correctly), categories are based on the description of the photos.
I was then looking how to update in bulk the description of several photos at once and, for instance, being able to add the name of the country or the city in the description so it becomes a category and be able to use that category to filter photos of a same country/city across all albums.
Which lead me to this topic, so I guess the function doesn't exist yet but I believe would be very useful.
Thanks!

**Reply by jbelien** [reply]
I also thought to add tags to the description to fit my needs ; apparently not yet possible but I've found #1481 that talks about this.

**Comment by dj-moss** [comment]
Any updates on this feature? Not being able to bulk edit descriptions makes it extremely difficult to add meaningful tags

**Comment by iwanttobefreak** [comment]
I don't know is the same wish. Edit bulk location and date for selection of multiple photos
